guides-and-tutorials/sbtc/how-to-run-sbtc-signer.md

```diff
@@ -0,0 +1,531 @@
+# Required: false
+# Environment: SIGNER_SIGNER__P2P__LISTEN_ON
+#
+# Defined in the provided docker compose, do not uncomment unless you know
+# what you're doing.
+# -------------------------------------------------------------------------
+listen_on = ["tcp://0.0.0.0:4122", "quic-v1://0.0.0.0:4122"]
+
+# The publicly accessible network endpoint to advertise to other nodes.
+#
+# If this is not specified then the node will attempt to use other peers on the
+# network to determine its public endpoint. This is the recommended
+# configuration for most users.
+#
+# If your network uses an advanced configuration with separate inbound/outbound
+# addresses then you must specify this value with your inbound address and
+# configure port-forwarding as auto-discovery will report your outbound address.
+#
+# Format: ["<protocol>:<ip>:<port>", ...] (see `listen_on` for protocol options)
+# Default: <none>
+# Required: false
+# Environment: SIGNER_SIGNER__P2P__PUBLIC_ENDPOINTS
+public_endpoints = []
+```
+
+</details>
+
+### Network Protocol Details
+
+The signer operates on port 4122 by default and supports both TCP and QUIC protocols for peer communication. The signer will attempt QUIC connections first for improved performance, automatically falling back to TCP if QUIC is unavailable or blocked on the network.
+
+## 3. Set up your blocklist client
+
+The blocklist client provides address screening services for the signer node. It interfaces with external API services to perform risk analysis on Bitcoin addresses. Default implementation uses Chainalysis, but supports custom implementations.
+
+Reference configuration:
+
+<details>
+
+<summary>blocklist-client-config.toml example</summary>
+
+```toml
+# !! ==============================================================================
+# !! Blocklist Client Configuration
+# !! ==============================================================================
+
+[server]
+# Server configurations.
+
+# The host the server will run on.
+host = "127.0.0.1"
+
+# The port the server will run on.
+port = 3032
+
+[risk_analysis]
+# Risk analysis configurations.
+
+# The URL of the API you're planning to use to assess confirming or denying a
+# bitcoin address. Note that the default implmentation of the blocklist client
+# assumes the use of Chainalysis, but any method can be used as long as the API
+# calls within the blocklist client are changed to match the new API.
+api_url = "https://api.chainalysis.com"
+
+# The API key for the API you're planning to use to assess confirming or denying.
+api_key = "your-api-key"
+```
+
+</details>
+
+## 4. Set up your container
+
+Specific instructions for running your Docker container will be added shortly, but this is an example of what the Docker compose file will look like, so you can begin to understand what you'll be running as a signer.
+
+Remember, this is not production-ready yet and is only for demonstration purposes at the moment.
```

Comment:
```suggestion
As we crystalize the Docker configuration, the following section displays an example of what the Docker compose file will look like, so that Signers can peek at how the system will look like.

Remember, this is not production-ready yet and is only for demonstration purposes at the moment.
```